charts/backstage/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.10.0
+version: 0.10.1
```

Comment:
wouldn't this be a minor increase? as we're adding the capability to specify target port? as opposed to patching a bug

Reply:
@ChrisJBurns fixed